IBPSA/Fluid/FixedResistances/Validation/PlugFlowPipes/Data/FlowRateGeneric.mo

```diff
@@ -0,0 +1,50 @@
+within IBPSA.Fluid.FixedResistances.Validation.PlugFlowPipes.Data;
```

Comment:
It seems like this dataset is never used in the current codebase. Should we include it in the merge?